src/Symfony/Component/HttpFoundation/File/MimeType/ExtensionGuesser.php

```diff
@@ -56,7 +56,7 @@ public static function getInstance()
     /**
      * Registers all natively provided extension guessers.
      */
-    private function __construct()
+    protected function __construct()
```

Comment:
@iltar thank you for a link and I see the point.

@fabpot this will be reverted as well.